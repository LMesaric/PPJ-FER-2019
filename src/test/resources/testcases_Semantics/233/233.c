int main(void) {
 int a[10];
 a[1]
 [2];
 return 0;
}


