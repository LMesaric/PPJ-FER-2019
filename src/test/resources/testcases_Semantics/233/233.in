<prijevodna_jedinica>
 <vanjska_deklaracija>
  <definicija_funkcije>
   <ime_tipa>
    <specifikator_tipa>
     KR_INT 1 int
   IDN 1 main
   L_ZAGRADA 1 (
   KR_VOID 1 void
   D_ZAGRADA 1 )
   <slozena_naredba>
    L_VIT_ZAGRADA 1 {
    <lista_deklaracija>
     <deklaracija>
      <ime_tipa>
       <specifikator_tipa>
        KR_INT 2 int
      <lista_init_deklaratora>
       <init_deklarator>
        <izravni_deklarator>
         IDN 2 a
         L_UGL_ZAGRADA 2 [
         BROJ 2 10
         D_UGL_ZAGRADA 2 ]
      TOCKAZAREZ 2 ;
    <lista_naredbi>
     <lista_naredbi>
      <naredba>
       <izraz_naredba>
        <izraz>
         <izraz_pridruzivanja>
          <log_ili_izraz>
           <log_i_izraz>
            <bin_ili_izraz>
             <bin_xili_izraz>
              <bin_i_izraz>
               <jednakosni_izraz>
                <odnosni_izraz>
                 <aditivni_izraz>
                  <multiplikativni_izraz>
                   <cast_izraz>
                    <unarni_izraz>
                     <postfiks_izraz>
                      <postfiks_izraz>
                       <postfiks_izraz>
                        <primarni_izraz>
                         IDN 3 a
                       L_UGL_ZAGRADA 3 [
                       <izraz>
                        <izraz_pridruzivanja>
                         <log_ili_izraz>
                          <log_i_izraz>
                           <bin_ili_izraz>
                            <bin_xili_izraz>
                             <bin_i_izraz>
                              <jednakosni_izraz>
                               <odnosni_izraz>
                                <aditivni_izraz>
                                 <multiplikativni_izraz>
                                  <cast_izraz>
                                   <unarni_izraz>
                                    <postfiks_izraz>
                                     <primarni_izraz>
                                      BROJ 3 1
                       D_UGL_ZAGRADA 3 ]
                      L_UGL_ZAGRADA 4 [
                      <izraz>
                       <izraz_pridruzivanja>
                        <log_ili_izraz>
                         <log_i_izraz>
                          <bin_ili_izraz>
                           <bin_xili_izraz>
                            <bin_i_izraz>
                             <jednakosni_izraz>
                              <odnosni_izraz>
                               <aditivni_izraz>
                                <multiplikativni_izraz>
                                 <cast_izraz>
                                  <unarni_izraz>
                                   <postfiks_izraz>
                                    <primarni_izraz>
                                     BROJ 4 2
                      D_UGL_ZAGRADA 4 ]
        TOCKAZAREZ 4 ;
     <naredba>
      <naredba_skoka>
       KR_RETURN 5 return
       <izraz>
        <izraz_pridruzivanja>
         <log_ili_izraz>
          <log_i_izraz>
           <bin_ili_izraz>
            <bin_xili_izraz>
             <bin_i_izraz>
              <jednakosni_izraz>
               <odnosni_izraz>
                <aditivni_izraz>
                 <multiplikativni_izraz>
                  <cast_izraz>
                   <unarni_izraz>
                    <postfiks_izraz>
                     <primarni_izraz>
                      BROJ 5 0
       TOCKAZAREZ 5 ;
    D_VIT_ZAGRADA 6 }
