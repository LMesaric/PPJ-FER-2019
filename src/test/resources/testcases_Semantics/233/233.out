<postfiks_izraz> ::= <postfiks_izraz> L_UGL_ZAGRADA(4,[) <izraz> D_UGL_ZAGRADA(4,])
